tpl/template_funcs.go

```diff
@@ -1699,13 +1699,21 @@ func countRunes(content interface{}) (int, error) {
 }
 
 // humanize returns the humanized form of a single word.
+// If the word given can be identified as an integer, the behavior
+// is to add the corresponding ordinal.
 // Example:  "my-first-post" -> "My first post"
+// Example:  "103" -> "103rd"
 func humanize(in interface{}) (string, error) {
 	word, err := cast.ToStringE(in)
 	if err != nil {
 		return "", err
 	}
-	return inflect.Humanize(word), nil
+
+	_, err = strconv.Atoi(word)
```

Comment:
Agree. This should be integers; that said I think it would be cleaner if you pull the integer check to the top and did a type assertion for the integer types. Aka: It looks funky to first convert to a string, then back to int when that was what you had in the first place. The downside is that there are lots of integer types, but we should have a helper for that (like cast.ToInt, but not as permissive ...